src/components/engines/cms/modules/media/MediaCard.tsx

```
@@ -39,34 +39,60 @@ export function MediaCard({
+    setMediaDimensions(dimensions);
+  }, []);
+
   return (
     <div 
       ref={cardRef}
-      className={`border rounded-lg overflow-hidden group cursor-pointer ${
+      className={`border rounded-lg overflow-hidden group cursor-pointer relative ${
         isSelected ? 'ring-2 ring-blue-500' : ''
       }`}
       onClick={handleSelect}
     >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)